boa/metadata.py

```diff
@@ -339,8 +332,8 @@ def get_hash_contents(self):
 
         # trim_build_only_deps(self, dependencies)
         dependencies = (
-            self.get_dependencies("build")
-            + self.get_dependencies("host")
+            self.get_dependencies("build") +
+            self.get_dependencies("host")
             # self.output.requirements["build"] + self.output.requirements["host"]
         )
         dependencies = {x.name for x in dependencies}
```

Comment:
Line https://github.com/mamba-org/boa/pull/42/files#diff-2e1b4be18dab7f82649cc72e80bcb0f1d9923bef8be4afc036600785e7e8d3feR356

It complains with `W605 invalid escape sequence '\s'`, any idea why?